packages/Webkul/Admin/src/Http/Controllers/Settings/CurrencyController.php

```diff
@@ -28,8 +28,11 @@ public function index()
         if (request()->ajax()) {
             return app(CurrencyDataGrid::class)->toJson();
         }
-
-        return view('admin::settings.currencies.index');
+        $currencyPosition = [
+            'left' => 'Left', 'left_with_space' => 'Left With Space',
+            'right' => 'Right', 'right_with_space' => 'Right With Space',
+        ];
```

Comment:
- All key values, should be in individual lines.

Reply:
Also, add translations.

Reply:
Both tasks have been completed. All key values are now on individual lines, and translations have been added.